resource_metadata_test.go

```diff
@@ -13,5 +13,8 @@ func TestListResourceDetailss(t *testing.T) {
 	if req.name() != "listResourceDetails" {
 		t.Errorf("API call doesn't match")
 	}
+	if req.description() != "List resource detail(s)" {
+		t.Errorf("API call doesn't match")
+	}
```

Comment:
this is gone if you rebase... at least it should be.

Reply:
ok i'll try to rebase

Reply:
I have rebased on master but that not resolve this issue